diff --git a/lib/dynamodb-utils.d.ts b/lib/dynamodb-utils.d.ts
index 4e8ce5422ea9..95a679e99964 100644
--- a/lib/dynamodb-utils.d.ts
+++ b/lib/dynamodb-utils.d.ts
@@ -5,7 +5,7 @@ export type DynamoDBStringSetResult = {
     SS: string[];
 };
 export type DynamoDBNumberResult = {
-    N: string;
+    N: number;
 };
 export type DynamoDBNumberSetResult = {
     NS: string[];
@@ -307,4 +307,4 @@ export type DynamodbUtil = {
      */
     fromS3ObjectJson(s3ObjectString: string): any;
 };
-//# sourceMappingURL=dynamodb-utils.d.ts.map
\ No newline at end of file
+//# sourceMappingURL=dynamodb-utils.d.ts.map
